app/models/accrediting_provider_enrichment.rb

```diff
@@ -0,0 +1,39 @@
+class AccreditingProviderEnrichment
+  include ActiveModel::Validations
+  include ActiveModel::Model
+
+  attr_accessor :UcasProviderCode, :Description
```

Comment:
Should these be `lower_snake_cased`?

Reply:
thats was how the original json stored as